src/coreclr/dlls/mscoree/mscorwks_ntdef.src

```diff
@@ -43,6 +43,7 @@ EXPORTS
         mono_assembly_foreach
         mono_assembly_get_image
         mono_assembly_get_object
+        mono_assembly_has_attribute
```

Comment:
Nit, if there are not in actual mono embedding interface, we may want `mono_unity_` prefix

Reply:
That's true. We can probably just drop mono altogether and have `unity_`?

Reply:
I would stick to whatever convention we are using. In Mono we did `mono_unity`. I see some `coreclr_` extra functions we added here.